dev_tools/auto_merge/README.md

```diff
@@ -0,0 +1,41 @@
+# Automerge bot 
+
+The automerge bot continuously watches a github repository for PRs labelled with the
+'automerge' label. When it sees such a PR it will mark it by labelling it with
+the 'front_of_queue_automerge' label. While there is an 'front_of_queue_automerge'
+labelled PR, the script will not label any other PRs with 'front_of_queue_automerge'.
+
+While there is a 'front_of_queue_automerge' labelled PR, the script will sync that PR
```

Comment:
It actually picks only one of the PRs to focus on. It prefers PRs that require less work (e.g. tests already passed) and then breaks ties using the lowest PR number.

Reply:
Thanks for pointing that out - clarified!